certified-connectors/Vocean/apiDefinition.swagger.json

```diff
@@ -719,7 +719,7 @@
             "x-ms-summary": "Activity type.",
             "enum": [
               "Innovate",
-              "Vote",
```

Comment:
@mialandin An enum value has been removed from the input field. Existing flows may have this value chosen and will continue to send it. can you please confirm that no users will be affected by this change?

Reply:
Hi @Amjed-Ayoub, 
As I also stated in the comment to the pull request: This will not affect our users. I have tested and confirmed that flows set up with the old enums will continue to work with the new enums. (We handle both the old and the new enums in our API, we just does not want the user to see the old enum.) Hope this is enough to reasure you that this change is ok. But, please, let me know if there's any other information you need to be certain.

The tests in short:
1. Set up flows with a custom connector matching the current certified connnector
2. verified these flows work with our production API
3. Updated the custom connector with the new enum setup
4. verified the flows still work with our production API
5. Set up new flows with the updated custom connector
6. verified that these also work with our production environment

Reply:
@mialandin thank you so much for the confirmation